### PR TITLE
fix isArray

### PR DESCRIPTION
**Pull request by Nemo64**
Array.isArray is not available in IE8.
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray#Browser_Unterst.C3.BCtzung

### CODE REVIEW

File: query-string.js [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 			if (!ret.hasOwnProperty(key)) {
 				ret[key] = val;
-			} else if (Array.isArray(ret[key])) {
+			} else if (ret[key] instanceof ret[key]) {
```
Comment (kevva) [inline query-string.js]: `ret[key] instanceof Array)`?

**Comment by kevva** [comment]
Not sure IE8 support is needed to be honest.

**Comment by sindresorhus** [comment]
No thanks

**Comment by Nemo64** [comment]
No thanks because you don't want ie8 support or no thanks because my
implementation is rushed because I quickly needed it?

Sindre Sorhus <notifications@github.com> schrieb am Di., 31. März 2015
17:07:

**Comment by kevva** [comment]
No thanks to IE8 support.

**Pull request closed without merging**